examples/esp32spi_simpletest.py

```diff
@@ -10,6 +10,19 @@
 TEXT_URL = "http://wifitest.adafruit.com/testwifi/index.html"
 JSON_URL = "http://api.coindesk.com/v1/bpi/currentprice/USD.json"
 
+
+# If you have an externally connected ESP32:
+esp32_cs = DigitalInOut(board.D9)
+esp32_ready = DigitalInOut(board.D10)
+esp32_reset = DigitalInOut(board.D5)
+
+# If you are using a board with pre-defined ESP32 Pins:
+"""
+esp32_cs = DigitalInOut(board.ESP_CS)
+esp32_ready = DigitalInOut(board.ESP_BUSY)
+esp32_reset = DigitalInOut(board.ESP_RESET)
+"""
+
 esp32_cs = DigitalInOut(board.ESP_CS)
```

Comment:
these override the lines above (lines 15-17)

Reply:
I think you meant to remove lines 26-28

Reply:
Yup, adding in on #952b38b, thanks for the catch!